v3/src/components/graph/adornments/movable-line/movable-line-adornment-model.ts

```diff
@@ -33,14 +72,33 @@ export const MovableLineAdornmentModel = AdornmentModel
   type: types.optional(types.literal(kMovableLineType), kMovableLineType),
   lines: types.map(MovableLineInstance)
 })
+.views(self => ({
+  get lineDescriptions() {
+    const lineDescriptions: ILineDescription[] = []
+    self.lines.forEach((line, key) => {
+      const { intercept, slope } = line.slopeAndIntercept
+      if (!isFinite(intercept) || !isFinite(slope)) return
+      const cellKey = JSON.parse(key)
+      lineDescriptions.push({ cellKey, intercept, slope })
+    })
+    return lineDescriptions
+  }
+}))
 .actions(self => ({
   setLine(
-    aLine: {intercept: number, slope: number, pivot1?: Point, pivot2?: Point, equationCoords?: Point}, key=''
+    aLine: {intercept: number, slope: number, pivot1?: Point, pivot2?: Point, equationCoords?: Point}, key=""
   ) {
     self.lines.set(key, aLine)
     const line = self.lines.get(key)
     line?.setPivot1(aLine.pivot1 ?? kInfinitePoint)
     line?.setPivot2(aLine.pivot2 ?? kInfinitePoint)
+  },
+  updateVolatileProps(
```

Comment:
```suggestion
  setVolatileLine(
```
I realize the implementation is somewhat different, but they're doing the same thing from the client's perspective.